packages/core/components/Puck/components/Fields/index.tsx

```diff
@@ -0,0 +1,199 @@
+import { ClipLoader } from "react-spinners";
+import { rootDroppableId } from "../../../../lib/root-droppable-id";
+import {
+  ReplaceAction,
+  SetAction,
+  replaceAction,
+  setAction,
+} from "../../../../reducer";
+import { Field, UiState } from "../../../../types/Config";
+import { InputOrGroup } from "../../../InputOrGroup";
+import { useAppContext } from "../../context";
+
+import styles from "./styles.module.css";
+import { getClassNameFactory } from "../../../../lib";
+import { ReactNode, useMemo } from "react";
+import { ItemSelector } from "../../../../lib/get-item";
+
+const getClassName = getClassNameFactory("PuckFields", styles);
+
+const defaultPageFields: Record<string, Field> = {
+  title: { type: "text" },
+};
+
+const DefaultFields = ({
+  children,
+  isLoading,
+}: {
+  children: ReactNode;
+  isLoading: boolean;
+  itemSelector?: ItemSelector | null;
+}) => {
+  return (
+    <div className={getClassName()}>
+      {children}
+      {isLoading && (
+        <div className={getClassName("loadingOverlay")}>
+          <ClipLoader />
```

Comment:
`aria-label="Loading"`

Reply:
Added to all ClipLoaders. Should probably spin this out to a component at some point.